### ISSUE TITLE
Found 2 vulnerabilities after installing module by npm

### ISSUE THREAD

**Issue by 314513535** [huiyan-fe/react-bmap#56]
Below is the [npm audit] details:

**Moderate**        Regular Expression Denial of Service
  Package         mime
  Patched in      >= 1.4.1 < 2.0.0 || >= 2.0.3
  Dependency of   react-bmap
  Path            react-bmap > kits-js > superagent > mime
  More info       https://nodesecurity.io/advisories/535

  **Low**             Large gzip Denial of Service
  Package         superagent
  Patched in      >=3.7.0
  Dependency of   react-bmap
  Path            react-bmap > kits-js > superagent
  More info       https://nodesecurity.io/advisories/479

found 2 vulnerabilities (1 low, 1 moderate) in 37672 scanned packages
  2 vulnerabilities require manual review. See the full report for details.

I tried to fix them by [npm audit fix] but only got below info:
fixed 0 of 2 vulnerabilities in 37672 scanned packages
  2 vulnerabilities required manual review and could not be updated

**Comment by Junior2Ran** [comment]
It seems the vulnerabilities are from `kits-js`.
add @danieltao1993 to check it out.

**Comment by 314513535** [comment]
Yes. The low severity one is from superagent package and the moderate one is from mime.
According to [superagent known vulnerability page](https://snyk.io/node-js/superagent) , The ReDoS was found but hasn't been fixed.

**Comment by Junior2Ran** [comment]
I see where the point is. `kits-js` uses `superagent@1.4.0` package([see it here](https://github.com/danieltao1993/kits-js/blob/master/package.json)), which version is tagged low severity on npm https://nodesecurity.io/advisories/479.

To solve the case, we can update `superagent` to version 3.7.0 or later in `kits-js`.

@danieltao1993 FYI.
